Resources/Private/Templates/Tools/Overview.html

```diff
@@ -1,4 +1,6 @@
-<html data-namespace-typo3-fluid="true" xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers">
+<html lang="en"
+      data-namespace-typo3-fluid="true"
```

Comment:
Always move `data-namespace-typo3-fluid` to last position. Why?
In most cases you have to add one or two new ViewHelper declarations. It's easier to mark the f namespace line and press CONTROL+D (duble the line).
In your solution I additionally have to move cursor to end of line and remove a duplicated `>`